ffi/src/lib.rs

```diff
@@ -30,7 +30,7 @@ pub enum FfiResult {
     Ok = 0_i32,
 }
 
-/// Implement [`Handle`] for given types with first argument as the initial handle id.
```

Comment:
@ales-tsurko this is the only link that is still unresolved.
I think my knowledge of Rust is limited here to understand why all the syntax I'm trying is failing, so if you have any ideas how to fix this -- please do

Reply:
The problem caused by the `Handle` trait defined inside macro, so there's no trait definition until the macro is used. We have two options: 
1. define the trait outside the macro;
2. don't link documentation for the trait.

@appetrosyan what do you think?

Reply:
maybe we can omit the link altogether? the description would still be clear even without the link

Reply:
> don't link documentation for the trait.

Is fine by me. We should avoid defining non-private traits inside macros, though.